test/e2e/testdata/istio/istio_csr_with_namespace_selector.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: operator.openshift.io/v1alpha1
```

Comment:
nit: Do we need to have different manifests for each scenario? Can we use one with bare minimum spec as base and add required configs in code?